util/reflect_exported_test.go

```diff
@@ -27,6 +29,30 @@ import (
 	"github.com/openconfig/ygot/util"
 )
 
+var (
+	PrintMapKeysSchemaAnnotationFunc = func(ni *util.NodeInfo, in, out interface{}) (errs util.Errors) {
+		if util.IsNilOrInvalidValue(ni.FieldKey) {
+			return
+		}
+		outs := out.(*string)
+		s := "nil"
+		if !util.IsNilOrInvalidValue(ni.FieldValue) {
+			s = pretty.Sprint(ni.FieldValue.Interface())
+		}
+
+		fn, err := util.SchemaPaths(ni.StructField)
+		if err != nil {
+			errs = append(errs, err)
+		}
+		if l := len(fn); l != 1 {
+			errs = append(errs, fmt.Errorf("invalid schema path length %d for %v", l, ni.StructField.Name))
+		}
+
+		*outs += fmt.Sprintf("%s/%s : \n%s\n, ", util.ValueStr(ni.FieldKey.Interface()), fn[0][0], util.ValueStr(s))
+		return
```

Comment:
if you never change value of the var, than make make  a normal func.

example https://go.dev/play/p/rKHcBI8Hpo4

Reply:
This is just copy-paste, let me fix this.